cypress/integration/functional/wallet/balances/convert/convert.spec.ts

```diff
@@ -3,32 +3,35 @@ context('Wallet - Convert DFI', () => {
     cy.createEmptyWallet(true)
 
     cy.getByTestID('bottom_tab_settings').click()
-    cy.sendDFItoWallet().sendDFItoWallet().sendDFITokentoWallet().wait(4000)
+    cy
+      // .sendDFItoWallet()
```

Comment:
@thedoublejay the main issue is here. It prompt twice for 2 utxos to sign, I just avoid it for now. Probably we need to do some upgrade on jellyfish side first (quite tricky)